sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_state.py

```diff
@@ -14,7 +14,8 @@
 _STATSBEAT_STATE_LOCK = threading.Lock()
 
 def is_statsbeat_enabled():
-    return not os.environ.get("APPLICATIONINSIGHTS_STATSBEAT_DISABLED_ALL")
+    disabled = os.environ.get("APPLICATIONINSIGHTS_STATSBEAT_DISABLED_ALL")
+    return disabled is None or disabled.lower() != "true"
```

Comment:
Not the biggest deal since I doubt this keyword will change, but since it's used here and in tests, I think we should use a constant.

Reply:
I think tests do not count as "used in multiple places". If we ever use this keyword somewhere else in the source code itself then we can think about putting it in constants.